lib/faker/bot/emoji_store/default.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module Faker
+  module Bot
+    module EmojiStore
+      DEFAULT = {
+        'Faker::Address' => ['🏡'],
+        'Faker::Alphanumeric' => ['🔠', '🔢'],
+        'Faker::Ancient' => ['📜'],
+        'Faker::App' => ['👩‍💻', '👨‍💻'],
+        'Faker::Appliance' => ['🎛', '🔌', '💡'],
+        'Faker::Artist' => ['👩‍🎨', '👨‍🎨', '🎨'],
+        'Faker::Avatar' => [],
+        'Faker::Bank' => ['💳', '🏦', '💸', '🏧', '💵'],
+        'Faker::Beer' => ['🍺', '🍻'],
+        'Faker::Boolean' => ['👍', '👎'],
+        'Faker::BossaNova' => ['🇧🇷', '🎤', '🎸'],
+        'Faker::Business' => ['💼', '📈', '📉', '📊'],
+        'Faker::Cannabis' => ['🍁'],
+        'Faker::ChileRut' => [],
+        'Faker::ChuckNorris' => ['🌵'],
+        'Faker::Code' => ['💻', '👾'],
+        'Faker::Coffee' => ['☕️'],
+        'Faker::Coin' => ['💰', '🤑', '💸', '💵'],
+        'Faker::Color' => ['🖌', '🎨', '🖼'],
+        'Faker::Commerce' => ['📈', '📉', '📊'],
+        'Faker::Company' => ['💼'],
+        'Faker::Compass' => ['🧭'],
+        'Faker::Construction' => ['🚧', '🏗', '👷', '👷‍'],
+        'Faker::Cosmere' => ['📚'],
+        'Faker::Crypto' => ['💰', '🤑', '💸', '💵'],
+        'Faker::CryptoCoin' => ['💰', '🤑', '💸', '💵'],
+        'Faker::Currency' => ['💰', '🤑', '💸', '💵'],
+        'Faker::Date' => ['📅'],
+        'Faker::DcComics' => ['🦇', '🦸‍', '🦸‍'],
+        'Faker::Demographic' => ['🛌', '🙎‍', '🙍‍♂️'],
```

Comment:
I think we do this in a different way in Faker. Check [this](https://github.com/faker-ruby/faker/blob/master/lib/faker/default/lorem.rb#L58) and [this](https://github.com/faker-ruby/faker/blob/master/lib/locales/en/lorem.yml#L7) out and let met know your thoughts.

Reply:
Aaah, interesting approach! I'll have a look. 🙂